zerver/lib/actions.py

```diff
@@ -1181,6 +1181,12 @@ def do_deactivate_stream(
     # Get the affected user ids *before* we deactivate everybody.
     affected_user_ids = can_access_stream_user_ids(stream)
 
+    Subscription.objects.filter(
+        recipient__type=Recipient.STREAM,
```

Comment:
Since this part handles the case of changing admin roles for deactivated streams, it doesn't exactly fall under "Unsubscribing stream admin" case as the commit indicates. Maybe want to update the title / move this to another commit?

Reply:
I have added this because deactivating a stream first unsubscribes all the members using this line `get_active_subscriptions_for_stream_id(stream.id).update(active=False)`. Maybe I can specify this in commit message description.